grails-app/services/au/org/ala/biocollect/merit/WebService.groovy

```diff
@@ -290,6 +327,11 @@ class WebService {
         }
     }
 
+    def deleteWrapper(url){
```

Comment:
Define the return type for doDelete and deleteWrapper so it's a bit more obvious why we have 2 methods (doDelete return int, deleteWrapper returns Map)